ext/zip/php_zip.stub.php

```diff
@@ -64,10 +64,25 @@ function zip_entry_filesize($zip_entry): int|false {}
  */
 function zip_entry_compressionmethod($zip_entry): string|false {}
 
-class ZipArchive
+class ZipArchive implements Countable
 {
+    /** @var int|null */
+    public $lastId;
+
+    /** @var int|null */
+    public $status;
+
+    /** @var int|null */
+    public $statusSys;
+
+    /** @var int|null */
+    public $numFiles;
+
+    /** @var string|null */
+    public $filename;
+
     /** @var string|null */
-    public $name;
+    public $comment;
```

Comment:
We should convert these to uninit typed props in a followup.

Reply:
Agreed! I'll soon create one for this and mysqli.

Actually, I wanted to ask you two slightly connected questions to this PR.
- Couldn't we declare the undeclared properties in ext/dom? (and I found another extension using dynamic properties when working on this PR)
- I want to backport the missing classes to the PHP-8.0 branch from this PR and the one for ext/spl. Should I include the stub file with the SPL exceptions as well, even though that file wouldn't be used at all...?

Reply:
> Couldn't we declare the undeclared properties in ext/dom? (and I found another extension using dynamic properties when working on this PR)

Sounds reasonable.

> I want to backport the missing classes to the PHP-8.0 branch from this PR and the one for ext/spl. Should I include the stub file with the SPL exceptions as well, even though that file wouldn't be used at all...?

Don't see a problem with that. Though TBH I'm no longer sure we need class declarations on PHP-8.0 at all.